### PR TITLE
[GraphQL] Update examples on GraphQL filtering page

### PR DESCRIPTION
**Pull request by crwaters16**

### CODE REVIEW

**Comment by crwaters16** [comment]
Curious if https://developers.cloudflare.com/analytics/graphql-api/features/filtering#graphql-003 also needs to change

**Comment by twalwyn** [comment]
> Curious if https://developers.cloudflare.com/analytics/graphql-api/features/filtering#graphql-003 also needs to change

That example is OK because it does not try to filter on a field that does not exist.

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 1]
```diff
@@ -104,8 +104,12 @@ The `like` operator is available for string comparisons and supports the `%` cha
 {
   myQuery(
```
Comment (twalwyn) [inline products/analytics/src/content/graphql-api/features/filtering/index.md]: This query was really messed up :)

This is an example query that will actually work: 

```suggestion
{
  viewer {
    zones(filter: {zoneTag: $zoneTag}) {
      httpRequestsAdaptiveGroups(filter: {datetime_gt: "2021-06-10T00:00:00Z", clientCountryName: "GB"}, limit: 1) {
        count
      }
    }
  }
}
```

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 2]
```diff
@@ -118,7 +122,7 @@ The following GraphQL example shows how to filter a specific node. The SQL equiv
 ##### GraphQL {#001}
 
 ```graphql
-httpRequests1hGroups(filter: {datetime: "2018-01-01T10:00:00Z"}) {
+httpRequestsAdaptiveGroups(filter: {datetime: "2018-01-01T10:00:00Z"}) {
```
Comment (twalwyn) [inline products/analytics/src/content/graphql-api/features/filtering/index.md:122]: The SQL below is incorrect for the given query. 

The SQL should read

```sql
WHERE datetime="2018-01-01T10:00:00Z"
```

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 3]
```diff
@@ -155,9 +159,10 @@ The following GraphQL example demonstrates using the `OR` operator in a filter.
 ##### GraphQL {#005}
 
 ```graphql
-httpRequests1hGroups(filter: {
-    datetime: "2018-01-01T10:00:00Z",
-    OR:[{clientCountryName: "US"}, {clientCountryName: "UK"}]) {
+httpRequestsAdaptiveGroups(
+        filter: {
+          datetime: "2018-01-01T10:00:00Z",
+          OR:[{clientCountryName: "US"}, {clientCountryName: "UK"}]) {
```
Comment (twalwyn) [inline products/analytics/src/content/graphql-api/features/filtering/index.md]:
```suggestion
          OR:[{clientCountryName: "US"}, {clientCountryName: "GB"}]) {
```
UK is not a country name in our analytics DB.

Comment (twalwyn) [reply products/analytics/src/content/graphql-api/features/filtering/index.md]: The SQL below should also probably be updated.

**Review by twalwyn — COMMENTED** [review]
Thanks for doing this cleanup work! I've suggested a few minor modifications.

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 4]
```diff
@@ -102,10 +102,22 @@ The `like` operator is available for string comparisons and supports the `%` cha
...
+      }
+    }
+  }
+}
     filter: {
-      clientCountry: "UK" # all objects having client country equal to "UK"
-      datetime_gt: "2018-01-01T10:00:00Z" # all object having datetime greater than "2018-01-01T10:00:00Z"
+      httpRequestsAdaptiveGroups(
+        filter: {
+          clientCountry: "UK" # all objects having client country equal to "UK"
```
Comment (crwaters16) [inline products/analytics/src/content/graphql-api/features/filtering/index.md]: @twalwyn, does this instance also need to change to "GB", or is that only for clientCountryName?

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 5]
```diff
@@ -102,10 +102,22 @@ The `like` operator is available for string comparisons and supports the `%` cha
 
 ```graphql
 {
-  myQuery(
+{
```
Comment (victor-perov) [inline products/analytics/src/content/graphql-api/features/filtering/index.md]: Lines 105-123 shows incorrect GraphQL query. I think it makes sense to just remove the general example section.

File: products/analytics/src/content/graphql-api/features/filtering/index.md (hunk 2 above)
Comment (victor-perov) [reply products/analytics/src/content/graphql-api/features/filtering/index.md:122]: So we should either change SQL (remove line 142) or add the country filter. For `httpRequestsAdaptiveGroups` a field responsible for a client's country would be `clientCountryName`.

Comment (victor-perov) [reply products/analytics/src/content/graphql-api/features/filtering/index.md:122]: I also don't see how this example differs from #005 below.

**Review by victor-perov — CHANGES_REQUESTED** [review]

**Review by kodster28 — APPROVED** [review]

File: products/analytics/src/content/graphql-api/features/filtering/index.md (hunk 5 above)
Comment (crwaters16) [reply products/analytics/src/content/graphql-api/features/filtering/index.md]: @victor-perov , think you may've reviewed around the same time as I implemented Tom's suggestions - let me know which of your comments stand with the new commits!

**Review by 888520ll19811209ll — APPROVED** [review]

**Review by 888520ll19811209ll — APPROVED** [review]

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 6]
```diff
@@ -102,10 +102,22 @@ The `like` operator is available for string comparisons and supports the `%` cha
 
 ```graphql
 {
```
Comment (twalwyn) [inline products/analytics/src/content/graphql-api/features/filtering/index.md:104]: Sorry if my original comment was not clear - L104 to L123 should be replace with the following:

```
{
  viewer {
    zones(filter: {zoneTag: $zoneTag}) {
      httpRequestsAdaptiveGroups(filter: {datetime_gt: "2021-06-10T00:00:00Z", clientCountryName: "GB"}, limit: 1) {
        count
      }
    }
  }
}
```

**Review by twalwyn — CHANGES_REQUESTED** [review]
Thanks for the update. I clarified one of my change requests.

**Review by twalwyn — APPROVED** [review]
LGTM!

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 7]
```diff
@@ -166,7 +167,7 @@ httpRequests1hGroups(filter: {
 
 ```sql
 WHERE datetime="2018-01-01T10:00:00Z"
-  AND ((clientCountryName = "UK") OR (clientCountryName = "US"))
+  AND ((clientCountryName = "GB") OR (clientCountryName = "US"))
```
Comment (soheiokamoto) [inline products/analytics/src/content/graphql-api/features/filtering/index.md]: I don't know if it matters, but maybe keep the order the same as above GraphQL (`US` then `GB`).

File: products/analytics/src/content/graphql-api/features/filtering/index.md [hunk 8]
```diff
@@ -155,9 +155,10 @@ The following GraphQL example demonstrates using the `OR` operator in a filter.
 ##### GraphQL {#005}
 
 ```graphql
-httpRequests1hGroups(filter: {
-    datetime: "2018-01-01T10:00:00Z",
-    OR:[{clientCountryName: "US"}, {clientCountryName: "UK"}]) {
+httpRequestsAdaptiveGroups(
+        filter: {
+          datetime: "2018-01-01T10:00:00Z",
+          OR:[{clientCountryName: "US"}, {clientCountryName: "GB"}]) {
```
Comment (soheiokamoto) [inline products/analytics/src/content/graphql-api/features/filtering/index.md:161]: There is still `UK` in the paragraph above.

Comment (victor-perov) [reply products/analytics/src/content/graphql-api/features/filtering/index.md:161]: Yep, would be nice to change line 153 as well

**Review by victor-perov — CHANGES_REQUESTED** [review]
Thanks for your work! It looks much better now.
Would you mind changing L153 to align countryName (UK -> GB) with the example?

**Review by victor-perov — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**